config/presets/testnet.go

```diff
@@ -65,11 +66,11 @@ func testnet() config.Config {
 	conf.Beacon.FirstVotingRoundDuration = 3 * time.Minute
 	conf.Beacon.GracePeriodDuration = 10 * time.Second
 	conf.Beacon.ProposalDuration = 30 * time.Second
+	conf.Beacon.VotingRoundDuration = 50 * time.Second
```

Comment:
why do you move stuff around?

Reply:
I diffed `testnet.go` and `fastnet.go` and to check the differences. `fastnet.go` is missing a few parameters that are set in `testnet.go`. I can revert these changes if you want.